starters/next-wordpress-starter/next.config.js

```diff
@@ -12,6 +12,9 @@ const nextConfig = {
   env: {
     backendUrl: process.env.WPGRAPHQL_URL,
   },
+  images: {
+    domains: [process.env.IMAGE_DOMAIN],
```

Comment:
Oh one more thing, can you add this env var to the `.env.example`?

Reply:
Sure